app/javascript/components/Login.jsx

```diff
@@ -20,14 +22,23 @@ class Login extends Component{
         });
     }
 
-    onSubmitForm(e){
-        
+    handleSubmit(e){
+        e.preventDefault();
+
+        this.onSubmitForm(e.target);
+    }
+
+    async onSubmitForm(e){
```

Comment:
Let's not make this a different function. We can just do the same thing we're doing here, but in the `handleSubmit` function.